Python/specialize.c

```diff
@@ -1712,19 +1769,19 @@ function_get_version(PyObject *o, int opcode)
     }
     return version;
 }
-#endif  // Py_GIL_DISABLED
 
 /* Returning 0 indicates a failure. */
 static uint32_t
 type_get_version(PyTypeObject *t, int opcode)
 {
-    uint32_t version = t->tp_version_tag;
+    uint32_t version = FT_ATOMIC_LOAD_UINT_RELAXED(t->tp_version_tag);
```

Comment:
I don't think this is needed: `type_get_version` is only used in the default build.
```suggestion
    uint32_t version = t->tp_version_tag;
```